test/test_helper.rb

```diff
@@ -1,8 +1,8 @@
 ENV['RAILS_ENV'] ||= 'test'
-require File.expand_path('../../config/environment', __FILE__)
+require File.expand_path('../../config/environment', __dir__)
 require 'rails/test_help'
 
-class ActiveSupport::TestCase
+class ActiveSupport < TestCase
```

Comment:
We will not use any of the test directory's files, you can delete the whole thing `test/`